src/index.ts

```diff
@@ -0,0 +1,19 @@
+import "./set-public-path";
+import { routeRegex } from "@openmrs/esm-root-config";
+import { backendDependencies } from "./openmrs-backend-dependencies";
+
+const importTranslation = require.context(
+  "../translations",
+  false,
+  /.json$/,
+  "lazy"
+);
+
+function setupOpenMRS() {
+  return {
+    lifecycle: () => import("./openmrs-esm-patient-chart"),
+    activate: location => routeRegex(/^patient\/.+\/chart/, location)
```

Comment:
Perhaps we can get rid of the single-spa dependency by taking advantage of strings as activation functions and lists of activation functions (since [single-spa 5.3.0](https://github.com/single-spa/single-spa/releases/tag/v5.3.0))?

```suggestion
    activate: ["patient-chart", "patient_chart"]
```

Reply:
That would be cool. Do these strings also work with our base route? We are not at "/", but rather at "/openmrs/spa".

Reply:
Oh dang, that's right... perhaps the initial script should do something like
```
           function preprocessActivator(activator) {
             if (Array.isArray(activator)) {
               return activator.map(preprocessActivator);
             } else if (typeof activator === "string") {
               return window.getOpenmrsSpaBase() + activator;
             } else {
               return activator
             }
           }
           registerApplication(appName, result.lifecycle, preprocessActivator(result.activate));
```

What do you think?

Reply:
Looks great to me!

I'll create a new PR for the openmrs-module-spa repo; I think the first approach with the `preprocessActivator` is the way to go.

I'd also include the translations from the openmrs-esm-root-config. Then its time to retire the repo. It served its purpose.